android/app/src/main/java/com/mattermost/rnbeta/NotificationsLifecycleFacade.java

```diff
@@ -79,6 +80,12 @@ public void onActivityResumed(Activity activity) {
         switchToVisible(activity);
 
         ReactContext ctx = getRunningReactContext();
+
+        if (ctx != null) {
+            ctx.getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class).
+                    emit(NATIVE_APP_LAUNCHED, null);
```

Comment:
will this event be emitted every time the app comes from the background? does it affect in any way?

Reply:
I'll check and see